src/space_flight/pipelines/data_engineering/nodes.py

```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
```

Comment:
Dude what is this?!

Reply:
![YOUR BAD CODE](https://user-images.githubusercontent.com/72718229/191439671-8b5e5699-27c8-4cca-b2ea-93c0e8634df1.png)

Reply:
it's code mate. Like, can't you this properly???

Reply:
What is I and J"?

Reply:
You blind ah?

Reply:
scott: “I can’t help it if you're too dumb to understand my leet code!”

Reply:
Rohit : The machine is able to understand my code, why can't you?

Reply:
Prateek: It works on my machine

Reply:
The machine is able to understand my code, why can't you

It works on my machine, it's you that's stupid.

I can’t help it if you're too dumb to understand my leet code!

Come see me in the car park later and i can explain it to you.